src/ScriptBuilder/Meters/MeterDefinitionReader.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace NServiceBus.Metrics.PerformanceCounters
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Mono.Cecil;
+
+    static class MeterDefinitionReader
+    {
+        public static bool TryGetMeterDefinition(TypeDefinition type, out List<MeterDefinition> definitions)
+        {
+            definitions = new List<MeterDefinition>();
+            if (type.BaseType != null && type.BaseType.FullName == "NServiceBus.Metrics.MetricBuilder")
+            {
+                var meterAttributes = type.Fields.Select(f => f.GetSingleAttribute("NServiceBus.Metrics.MeterAttribute")).Where(c => c != null);
+
+                foreach (var meterAttribute in meterAttributes)
+                {
+                    var name = (string)meterAttribute.ConstructorArguments[0].Value;
```

Comment:
Yeah, basically what we wanted is to avoid coupling this code to the indexes of the constructor arguments. So your PR solves that issue, thank you!